src/language/en-US.json

```
@@ -1037,5 +1037,6 @@
     "click_to_retry": "click to retry",
     "current_node_is_not_available": "Current node is not available. Trying to connect to node #{0}/{1}",
     "nodes_unavailable_check_network_and": "Nodes({0}/{1}) are currently unavailable. Please check your network connection, settings and ",
-    "copyright": "Copyright © 2021 NEM Software"
+    "copyright": "Copyright © 2021 NEM Software",
+    "address_list_cannot_be_empty": "Selected accounts cannot be empty. Please, select 1 or more accounts from the list."
```

```suggestion
    "address_list_cannot_be_empty": "Please, select 1 or more accounts from the list."
```

I think this sentence on its own is enough. Otherwise, it sounds like "the accounts are empty", as in, they have no funds.